src/Strategist/Strategist.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include <BWAPI.h>
+#include <queue>
+#include "./BuildOrders/DefaultBuildOrders.h"
```

Comment:
these `BuildOrder.h` includes should be in `Strategist.cpp`, they aren't used in the header at all.